content/faq.md

```diff
@@ -0,0 +1,157 @@
+
+For examples of how incentives can warp validation behavior, read about [miner extractable value (MEV)](https://arxiv.org/abs/1904.05234).
+
+
+#### Can financial institutions set up transaction validators to help them meet specific institutional standards and requirements?
+
+No, institutions cannot set up customized validator policies for choosing to allow some transactions and reject others. Validators either follow the protocol, or they do not. If software does not follow protocol rules, it does not function. Thus, it is not recommended that institutions seek out custom implementations without in-house expertise.
+
+
+#### What happens if more than 20% of nodes within the network do not agree with the majority? How is the final version of the ledger chosen?
+
+Normally, if there is a dispute about the validity of one transaction, that transaction gets pushed back until the majority can come to an agreement. But if more than 20% of the network did not follow the same protocol rules as the majority, the network would temporarily halt. It could resume when participants reconfigure their UNLs based on those who want to reach consensus with each other. This temporary processing delay is desired rather than double spending.
+
+In the process of determining the authoritative version of a ledger, there may be multiple temporary internal versions. Such internal versions naturally happen in distributed systems because not all nodes receive transactions in the same order. The analogous behavior in Bitcoin is where two servers each see a different longest chain because two blocks were mined at about the same time.
+
+However, there can be only one latest _validated_ ledger version at any given time; other versions are irrelevant and harmless.
+
+For more information about how the XRP Ledger's consensus mechanism behaves in adverse situations, see [Consensus Protections Against Attacks and Failure Modes](consensus-protections.html).
+
+
+#### Does the XRP Ledger have a formal process for adding validators?
+
+No, a formal process for adding validators is not compatible with the XRP Ledger, because it is a system with no central authority.
+
+Publishers of individual default UNLs set their own policies for when to add or remove validators from their lists of recommendations.
+
+For recommendations and best practices, see [Run `rippled` as a Validator](run-rippled-as-a-validator.html).
+
+
+## Role of XRP
+
+
+#### Why does Ripple use XRP holdings?
+
+Ripple's XRP holdings incentivize the company to make the XRP Ledger as useful as possible. XRP exists as a native asset in the XRP Ledger for anti-spam transaction purposes, and for currency bridging only if beneficial to users. Otherwise, XRP is completely optional.
```

Comment:
```suggestion
Ripple's XRP holdings incentivize the company to make the XRP Ledger as useful as possible. XRP exists as a native asset in the XRP Ledger for anti-spam transaction purposes, and for currency bridging only if beneficial to users. Otherwise, XRP is optional.
```